geomesa-hbase/geomesa-hbase-datastore/pom.xml

```diff
@@ -102,4 +87,35 @@
         </dependency>
     </dependencies>
 
+    <!-- pull in the coprocessor implementation for tests -->
+    <profiles>
+        <profile>
+            <id>hbase2</id>
+            <activation>
+                <!-- this will be deactivated if another profile in this block (i.e. hbase1) is activated -->
```

Comment:
Really?!  That's surprising to me.

Reply:
that's why i added a comment ;)
`activeByDefault` is screwy in maven, but it actually works well for this use case: https://maven.apache.org/guides/introduction/introduction-to-profiles.html